assets/scripts/admin/listresponse.js

```diff
@@ -140,6 +140,9 @@ function onDocumentReadyListresponse() {
         $('#change-display-mode-form').find('input[type=submit]').trigger('click');
     });
 
+    $(document).on("change", '#pageSize', function () {
```

Comment:
Just adding changePageSize class ?

Or update https://github.com/LimeSurvey/LimeSurvey/blob/bc809236a180e79f06b02c580b4435b0ad830605/application/extensions/LimeGridView/LimeGridView.php#L79 to use `data-action` or something else ?